tools/.gitignore

```diff
@@ -7,5 +7,9 @@ mkldscript
 reloc_prereq
 vtxdis
 yaz0
+gzinject/gzinject
+gzinject/obj/*
+z64compress/z64compress
+z64compress/o/*
```

Comment:
I assume this is to ignore the things that gets build on setup?

Reply:
you are correct